ideascube/widgets.py

```diff
@@ -1,6 +1,56 @@
 from django.forms import widgets
 
 from ideascube.configuration import get_config
+from django.utils.html import mark_safe, escape
+from django.utils.encoding import force_text
+from django.forms.utils import flatatt
+
+
+_script_template = '''<script type="text/javascript">tinymce.init({{
```

Comment:
No, we want to have on `{` in the result string and so we have to escape it to avoid `format` to handle it.